core/cache/cachetest/controller.go

```diff
@@ -95,6 +98,11 @@ func (tc *TestController) UpdateMachine(c *gc.C, modelUUID string, machine *stat
 	tc.SendChange(MachineChange(c, modelUUID, machine))
 }
 
+// UpdateUnit updates the input state unit in the cache.
+func (tc *TestController) UpdateUnit(c *gc.C, modelUUID string, unit *state.Unit) {
+	tc.SendChange(UnitChange(c, modelUUID, unit))
+}
```

Comment:
Nothing seems to call this function.

Reply:
Yes, I removed code that previously called it. I anticipate future use.